<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>locationManagerMainWindow</class>
 <widget class="QMainWindow" name="locationManagerMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>937</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Location Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>591</width>
      <height>491</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tableTab">
     <attribute name="title">
      <string>TableView</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="graphTab">
     <attribute name="title">
      <string>GraphView</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="mapTab">
     <attribute name="title">
      <string>MapView</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>110</y>
        <width>331</width>
        <height>161</height>
       </rect>
      </property>
      <property name="text">
       <string>This is going to contain a map of the selected trip</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>28</y>
      <width>304</width>
      <height>201</height>
     </rect>
    </property>
    <property name="firstDayOfWeek">
     <enum>Qt::Monday</enum>
    </property>
    <property name="gridVisible">
     <bool>true</bool>
    </property>
    <property name="horizontalHeaderFormat">
     <enum>QCalendarWidget::ShortDayNames</enum>
    </property>
    <property name="verticalHeaderFormat">
     <enum>QCalendarWidget::NoVerticalHeader</enum>
    </property>
    <property name="navigationBarVisible">
     <bool>true</bool>
    </property>
    <property name="dateEditEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QListView" name="tripViews">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>301</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>937</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuImport">
    <property name="title">
     <string>Import</string>
    </property>
    <addaction name="actionFrom_File"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuImport"/>
  </widget>
  <action name="actionFrom_File">
   <property name="text">
    <string>From File ...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>activated()</signal>
   <receiver>locationManagerMainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>397</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
